src/commands/start-daemon.command.ts

```diff
@@ -21,15 +21,43 @@ export interface Daemon {
 export const startDaemon = async (): Promise<void> => {
     process.chdir(findConfigDir());
     const pmConfigFileName = "dev-pm.config.js";
-    const { scripts: scriptDefinitions }: Config = await import(`${process.cwd()}/${pmConfigFileName}`);
-    const scripts = scriptDefinitions.map((scriptDefinition, id) => {
-        return new Script({ ...scriptDefinition, id });
-    });
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { scripts: scriptDefinitions }: Config = require(`${process.cwd()}/${pmConfigFileName}`);
     const daemon: Daemon = {
-        scripts,
+        scripts: scriptDefinitions.map((scriptDefinition, id) => {
+            return new Script({ ...scriptDefinition, id });
+        }),
         server: undefined,
     };
 
+    watchFile(`${process.cwd()}/${pmConfigFileName}`, async () => {
+        console.log(`${pmConfigFileName} file changed, reloading scripts`);
+        delete require.cache[require.resolve(`${process.cwd()}/${pmConfigFileName}`)];
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const { scripts: scriptDefinitions }: Config = require(`${process.cwd()}/${pmConfigFileName}`);
```

Comment:
This seems like a workaround to me. Alternatively, we could read the config using cosmiconfig: https://www.npmjs.com/package/cosmiconfig

Reply:
we could think about switching to comiconfig, but that is out of scope for this PR.

(I'm not sure cosmiconfig with all those different usage options is a good idea; see also how eslint changed [their config](https://eslint.org/blog/2023/10/flat-config-rollout-plans/) (not related to cosmiconfig but still they had many options/formats and reduced that))

Reply:
I just hat a look at how codegen does it, and they're using cosmiconfig. Let's defer this for later.